### PR TITLE
Changes Some Job Hud Sprites

### PR DESCRIPTION
**Pull request by nmajask**
# General Documentation

### Intent of your Pull Request

Changes Paramedic, Psychiatrist, Signal Technician, Tourist, and Clerk hud sprites to be more consistent with other hud sprites, here are the new ones:
Paramedic Icon
Psychiatrist
Signal Technician
tourist
clerk

### Why is this change good for the game?

I feel the new sprites fit the game and other HUD sprites better, but it is personal preference 

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 

It changes some sprites to look better

### What should players be aware of when it comes to the changes your PR is implementing?

Some hud sprites will change a bit

### What general grouping does this PR fall under? 

Sprite Changes

### Are there any aspects of the PR that you would like us not to mention on the Wiki?

No

### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 

Also no

# Changelog

:cl:    
imageadd: added some new hud icons  
imagedel: deleted some of the old hud icons 
/:cl:

### CODE REVIEW

**Review by nesisknesis — CHANGES_REQUESTED** [review]
Itd probably not bother most players, but the shading on the Paramedic icon looks a little blurry in game with the way youve shaded it, and I think i've got a better design for a the Sig Tech icon that flows more like a radio signal symbol. The Clerk icon though is pretty uninspired.
image

Here's my edits of them. I've used solely flat colour so that its more plain and bold, and the Clerk icon i've reverted back to the old.
image
raw:
hudclerk
hudpara
hudtech


The tourist icon is really good, lot more creative than the camera which was pretty ambiguous itself.

If you apply my edits i'll merge asap

**Comment by nmajask** [comment]
Thanks for the feedback, I am new to doing anything with Byond, textures, and GitHub so sorry if something is very bad. The sigtech and paramed icons you gave are way better than the ones I made, as both were just recolors of other sprites but with shading.

 Originally, I had just planned on making the clerk icon the same as the one you gave, but wanted to try changing it to a C as I saw the assistants icon used it and didn't know what else to do, but in hindsight the C is a bad idea as, unlike the assistant, there are other jobs, even within the same department, that start with a C and is also not very creative. I still feel like the one you provided is not very good, but I have know clue on what would be a better original sprite. You could use the cargo tech icon and recolor it to grey, but I don't know.

Anyways, I have applied your requested changes and checked in a test server if it works, and everything looks good. Once again, thanks for the feedback.

**Review by nesisknesis — APPROVED** [review]
Nothing is bad, these changes are very good.

Whoever made the assistant icon was probably equally perplexed as to what icon it should have had. At least with clerk we have a pre-established symbol.
